src/store/store.ts

```diff
@@ -76,7 +76,8 @@ export const useStore = defineStore('store', {
     isMobile (): boolean {
     // fall back to base window width if no window size changes have occurred
       const width = this.stateModel.windowWidth || window.innerWidth
-      return (width < new Vuetify().framework.breakpoint.thresholds.sm)
+      return width < 333
+      // return (width < new Vuetify().framework.breakpoint.thresholds.sm)
```

Comment:
This needs to be fixed for sure. We can't call Vuetify() anymore with Vuetify 3.

Reply:
Maybe we can use something like this?

https://vuetifyjs.com/en/features/display-and-platform/#examples

Reply:
You fixed it. Resolving.